src/boot/efi/log.h

```diff
@@ -19,13 +19,46 @@ __attribute__((no_stack_protector, noinline)) void __stack_chk_guard_init(void);
 #  define __stack_chk_guard_init()
 #endif
 
+typedef enum {
+        LOG_FATAL,
+        LOG_ERROR,
+        LOG_WARNING,
+        LOG_INFO,
+        LOG_DEBUG,
+        LOG_TRACE,
+} LogLevel;
```

Comment:
hmm, I think it would be nice if we'd stick to syslog levels, even if they are a bit weird.

Reply:
(or at least a reasonable subset of the BSD syslog levels, wit the same values as the real ones)